### ISSUE TITLE
any RailsView user has full access to all API endpoints

### ISSUE THREAD

**Issue by mkristian** [buzzn/core#661]
currently we have only two clients using the API endpoints: the powertaker form SPA and the some bubble-charts from the RailsView.

the powertaker-form SPA one needs to be public (maybe not now but in the long run), that is OK as it is.

the RailsView currently uses an oauth scope which gives access to the complete API, i.e. inviting anyone to probe our permission implementation and look for information leaks. that can lead to severe security breaches.

for development of new SPA we have the staging server and there we can allow admins to use the `full` scope for development. once a new SPA is ready we should add a scope for this SPA as a first step. the allowed scope should also depends on the life-cycle status of the user, etc

having a `full` scope on production creates a door into the system, it has a lock but this lock can be broken or an admin looses his/her key to the lock. we still have the rails console if we need to dig deeper into the database.
